views/js/component/eligibilityEditor.js

```diff
@@ -254,7 +255,20 @@ define([
                     deliveryTreeId : deliveryTreeId,
                     deliveryName : deliveryName
                 })
-                .render($container);
+                .render($container)
+                .resize();
+            },
+
+            /**
+             * Changes eligibilityEditor modal height and adds scroll for the modal to fit the screen height
+             *
+             * @returns {eligibilityEditor} chains the component
+             */
+            resize: function resize() {
+                eligibilityEditor.getElement().css('top', '40px');
+                eligibilityEditor.getElement().css('max-height', ($(window).height() - 80) + 'px');
+                eligibilityEditor.getElement().css('overflow-y', 'scroll');
```

Comment:
It should be better to gather all of this CSS changes into one single call, unless each command is dependent to the previous:
```javascript
eligibilityEditor.getElement().css({
    'top': '40px',
    'max-height': ($(window).height() - 80) + 'px',
    'overflow-y': 'scroll'
});
```

Reply:
Agree, will move it.